modules/lib/lib-project/src/test/resources/test/ModifyProjectHandlerTest.js

```diff
@@ -0,0 +1,133 @@
+var assert = require('/lib/xp/testing.js');
+var project = require('/lib/xp/project.js');
+
+var modifyProjectExpected = {
+    name: 'myproject',
+    displayName: 'new display name',
+    description: 'new description',
+    language: 'fr',
+    permissions: {
+        owner: [
+            'user:system:owner2',
+            'user:system:owner1'
+        ],
+        viewer: [
+            'user:system:viewer1'
+        ]
+    },
+    readAccess: {
+        public: true
+    }
+};
+
+exports.modifyProject = function () {
+    createProject();
+
+    var result = project.modify({
+        name: 'myproject',
+        displayName: 'new display name',
+        description: 'new description',
+        language: 'fr'
+    });
+
+
+    assert.assertJsonEquals(modifyProjectExpected, result);
+};
+
+var modifyDescriptionExpected = {
+    name: 'myproject',
+    displayName: 'project display name',
+    description: 'new description',
+    permissions: {
+        owner: [
+            'user:system:owner2',
+            'user:system:owner1'
+        ],
+        viewer: [
+            'user:system:viewer1'
+        ]
+    },
+    readAccess: {
+        public: true
+    }
+};
+
+exports.modifyDescription = function () {
+    createProject();
+
+    var result = project.modify({
+        name: 'myproject',
+        description: 'new description'
+    });
+
+    assert.assertJsonEquals(modifyDescriptionExpected, result);
+};
+
+var modifyDisplayNameExpected = {
+    name: 'myproject',
+    displayName: 'new display name',
+    description: 'project description',
+    permissions: {
+        owner: [
+            'user:system:owner2',
+            'user:system:owner1'
+        ],
+        viewer: [
+            'user:system:viewer1'
+        ]
+    },
+    readAccess: {
+        public: true
+    }
+};
+
+exports.modifyDisplayName = function () {
+    createProject();
+
+    var result = project.modify({
+        name: 'myproject',
+        displayName: 'new display name'
+    });
+
+    assert.assertJsonEquals(modifyDisplayNameExpected, result);
+};
+
+var modifyLanguageExpected = {
+    name: 'myproject',
+    displayName: 'project display name',
+    description: 'project description',
+    language: 'no',
+    permissions: {
+        owner: [
+            'user:system:owner2',
+            'user:system:owner1'
+        ],
+        viewer: [
+            'user:system:viewer1'
+        ]
+    },
+    readAccess: {
+        public: true
+    }
+};
+
+exports.modifyLanguage = function () {
+    createProject();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['createProject' was used before it was defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5511138)